specification/columns/chargeclass.md

```diff
@@ -0,0 +1,40 @@
+# Charge Class

```

Comment:
A few other possible names to hopefully spark ideas:
- Charge Context
- Charge Purpose
- Charge Reason